css/style.css

```diff
@@ -0,0 +1,51 @@
+.graybackground{
+  background-color: rgb(234, 229, 229);
+}
+main{
+  display: block;
+  float:right;
+}
+#main-doc{
+  line-height: 150%;
+  position: absolute;
+  margin-left: 310px;
+}
+
+#navbar {
+  position: fixed;
+  min-width: 290px;
+  top: 0px;
+  left: 0px;
+  width: 300px;
+  height: 100%;
+  background-color:blanchedalmond;
+  border-right: dashed;
+  border-color: rgba(0, 22, 22, 0.4);
+}
+nav{
```

Comment:
Please make sure to have one space between the name of the tag and the subsequent opening curly brace so as to pass linter checkers.